### ISSUE TITLE
Right channel is never used

### ISSUE THREAD

**Issue by alixaxel** [bbc/waveform-data.js#42]
On https://github.com/bbc/waveform-data.js/blob/master/lib/builders/audiodecoder.js#L38-L39 it should read:

```js
left_channel = audio_buffer.getChannelData(0);
right_channel = audio_buffer.getChannelData(1);
```

Also, the assumption that there are always 2 channels is flawed. Something like this would be more robust:

```js
var channels = [];

for (var i = 0; i < audio_buffer.numberOfChannels; ++i) {
	channels[i] = audio_buffer.getChannelData(i);
}

for (var i = 0; i < audio_buffer.length; ++i) {
	var sample = 0;

	for (var channel = 0; channel < channels.length; ++channel) {
		sample = channels[channel][i];
	}

	sample = sample / channels.length * scale_adjuster;
}
```

Or even:

```js
var channels = [];

for (var i = 0; i < Math.min(2, audio_buffer.numberOfChannels); ++i) { 
	channels[i] = audio_buffer.getChannelData(i);
}
```

To just consider stereo channels.

**Comment by chrisn** [comment]
Good catch, we'll get this fixed soon.

**Comment by chrisn** [comment]
@alixaxel I've just pushed a change to master that should fix this. Please take a look and let me know if this is OK. Thanks!

**Comment by chrisn** [comment]
This is fixed in version 2.0.1.
